odoh_server.go

```diff
@@ -116,15 +118,25 @@ func main() {
 	log.Printf("Setting Server Name as %v", serverName)
 
 	var experimentID string
-	if experimentID := os.Getenv(experimentIDEnvironmentVariable); experimentID == "" {
+	if experimentID = os.Getenv(experimentIDEnvironmentVariable); experimentID == "" {
```

Comment:
I'm not sure what the effect of this will be. Normally the `if <assignment>; <condition> { <scope> }` syntax is used to create and initialize a variable that's used only in the scope of the conditional. I'm not sure if `experimentID` has the value it should have on line 124.

To make sure this does what you think it should do, unambiguously, I would do:
```
experimentID := os.Getenv(experimentIDEnvironmentVariable)
if experimentID == "" {
  // Handle empty experiment ID
}
```

Here and below.

Reply:
This pattern should work fine, so I think this one just comes down to a style preference.